tests/cypress/latest/e2e/capa_eks_clusterclass.spec.ts

```diff
@@ -0,0 +1,123 @@
+import '~/support/commands';
+import {qase} from 'cypress-qase-reporter/mocha';
+import {getClusterName, skipClusterDeletion} from '~/support/utils';
+import {capaResourcesCleanup, capiClusterDeletion, importedRancherClusterDeletion} from "~/support/cleanup_support";
+
+Cypress.config();
+describe('Import CAPA EKS Class-Cluster', {tags: '@full'}, () => {
+  const timeout = 1200000
+  const classNamePrefix = 'aws-eks'
+  const clusterName = getClusterName(classNamePrefix)
+  const turtlesRepoUrl = 'https://github.com/rancher/turtles'
+  const classesPath = 'examples/clusterclasses/aws/eks'
+  const clusterClassRepoName = 'aws-eks-clusterclass'
+  const providerName = 'aws'
+  const accessKey = Cypress.env('aws_access_key')
+  const secretKey = Cypress.env('aws_secret_key')
+
+  beforeEach(() => {
+    cy.login();
+    cy.burgerMenuOperate('open');
+  });
+
+  it('Setup the namespace for importing', () => {
+    cy.namespaceAutoImport('Disable');
+  })
+
+  // TODO: Create Provider via UI, ref: capi-ui-extension/issues/128
+  it('Create AWS CAPIProvider & AWSClusterStaticIdentity', () => {
+    cy.removeCAPIResource('Providers', providerName);
+    cy.createCAPIProvider(providerName);
+    cy.checkCAPIProvider(providerName);
+    cy.createAWSClusterStaticIdentity(accessKey, secretKey);
+  })
+
+  qase(129,
+    it('Add CAPA EKS ClusterClass Fleet Repo', () => {
+      cy.addFleetGitRepo(clusterClassRepoName, turtlesRepoUrl, 'aws-eks-example', classesPath, 'capi-classes')
```

Comment:
```suggestion
      cy.addFleetGitRepo(clusterClassRepoName, turtlesRepoUrl, 'main', classesPath, 'capi-classes')
```
To be done post r/turtles PR merge